reactions/src/SPT.cpp

```diff
@@ -843,6 +908,89 @@ for(int zi = 0; zi<noz; zi++){
 }
 }
 
+// BILL MOD
+void SPT::ploop_init_nu(double ploopr[], double ploopp[], double redshifts[], int noz, double vars[], double k0){
+  IOW iow;
+
+  double res[noz],resp[noz]; // stores loop integral results for real and pseudo spectra
+  const int loop_N = 50; // steps in trap rule for loop integral in integrated wave vector p
```

Comment:
have we checked if these steps are enough for the precision needed?

Reply:
I've checked that it gives sub percent accuracy but it could be chosen to be less. I've added a comment.